controlsfx/src/main/java/org/controlsfx/validation/decoration/GraphicValidationDecoration.java

```diff
@@ -99,14 +111,36 @@ protected Node createDecorationNode(ValidationMessage message) {
         return label;
     }
 
+    protected Node getGraphicBySeverity(Severity severity) {
+        switch (severity) {
+        case ERROR:
+            return createErrorNode();
+        case WARNING:
+            return createWarningNode();

```

Comment:
`createErrorNode` and `createWarningNode` are deprecated. May be we should avoid using the same?

Reply:
Well in my former commit I actually avoided it, but then thought that it is not backwards compatible any more, because I actually added the @deprecated annotation to these methods.
If I still call these methods in our implementation it won't break existing code.
I don't know which rules you have when to remove/delete @deprecated methods,  but until then I think we should still call the @deprecated methods until we decide to remove these methods.

Agree?

Reply:
Sounds good. I was just being precautious :)